test.properties

```diff
@@ -3592,6 +3592,7 @@
     test.batch.class.names.excludes[acceptance]=\
         **/batch-planner/**/*Test.java,\
         **/headless-builder/**/*Test.java,\
+        **/object-rest-test/**/ExportTaskResourceTest.java,\
```

Comment:
@GaborKomaromi wrong test. This is the right one:

```
com.liferay.headless.batch.engine.resource.v1_0.test.ExportTaskResourceTest
```

Reply:
oh shhh. didn't realise that, thanks for noticing @4lejandrito!

Reply:
Changed to the right one.